src/tests/ftest/util/nvme_utils.py

```diff
@@ -120,7 +120,8 @@ def start_ior_thread(self, create_cont, operation):
 
         # Created new container
         if create_cont:
-            self.create_cont()
+            self.container = self.get_container(
+                self.pool, oclass=self.ior_local_cmd.dfs_oclass.value)
```

Comment:
What I don't like about this one is that it's _**adding**_ a `self.container` reference. Ideally, it should use a local container variable, but that requires some other work.

Reply:
From Phil, maybe this makes more sense.